grantnav/frontend/templatetags/frontend.py

```diff
@@ -195,8 +195,11 @@ def get_dataset(grant):
 
 @register.filter(name='get_current_sort')
 def get_current_sort(query):
-    for key, value in query['sort'].items():
-        return key + " " + value["order"]
+    if query:
+        for key, value in query['sort'].items():
+            return key + " " + value["order"]
```

Comment:
I think it's preferred to use fstrings here, this helps protect against the case where and int or other type needs coercion into a string.  Something like
```python
return f"{key} {value['order']}" 
````

Reply:
I only added the if/else block. I suspect it's flagged the rest on line changes. I suspect the existing concat pattern is consistent across a lot of grantnav but I agree the fstring approach is better. Should we opt for incremental improvements or consistency?